.gitignore

```diff
@@ -23,3 +23,16 @@ target/
 
 # data files
 .data/
+
+data/download/*
+!data/download/.gitkeep
+
+data/convert/*
+!data/convert/.gitkeep
+
+# TODO: move to under data folder
```

Comment:
TODO: remove TODO note

Reply:
a good example of a glitch in the Matrix.